packages/ui/product/Star.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+
+interface SvgProps {
+  fill: string;
+}
+
+const Star: FC<SvgProps> = ({ fill }) => {
```

Comment:
Star component should accept value

Reply:
added the prop to Rating and set to readOnly. Renamed to StarSVG

changes in [1bf879c](https://github.com/dandalandone/POC-EyCommerce/pull/36/commits/1bf879c0e00e42d36a2ca52da0cf408e1490207d)